src/main/scala/com/microsoft/hyperspace/actions/RefreshAppendAction.scala

```diff
@@ -60,25 +59,6 @@ class RefreshAppendAction(
     }
   }
 
-  private lazy val appendedFiles = {
```

Comment:
Note for reviewers: this method is moved to RefreshActionBase, and `IndexLogEntry.appendedFiles` is added to the result.

Reply:
thanks @apoorvedave1 , noted!